pkg/apis/componentconfig/edgecore/v1alpha1/default.go

```diff
@@ -283,3 +286,31 @@ func NewEdgeClusterEdgeCoreConfig() *EdgeCoreConfig {
 		},
 	}
 }
+
+//SetKubeconfigArray returns an array of kubeconfig files
+func SetKubeconfigArray() []string {
```

Comment:
SetKubeconfigArray -> SetKubeConfigArray

Reply:
Updated name of the function, thanks